evalml/automl/automl_search.py

```diff
@@ -581,6 +591,8 @@ def _find_best_pipeline(self):
                     X_train, X_threshold_tuning, y_train, y_threshold_tuning = split_data(X_train, y_train, self.problem_type,
                                                                                           test_size=0.2,
                                                                                           random_seed=self.random_seed)
+                if getattr(self.data_splitter, "transform", None):
+                    X_train, y_train = self.data_splitter.transform(X_train, y_train)
```

Comment:
Hmm, so we're applying this transformation in `_find_best_pipeline` but not during training?

Reply:
@dsherry, both the `split` and `transform` method resample the data. `transform` just doesn't also handle the splitting.